packages/ui/src/styles/foundations/fonts.ts

```diff
@@ -10,6 +10,6 @@
 // PARTICULAR PURPOSE. See the GNU General Public License for more details.
 
 export const fonts = {
+    heading: '"FK Grotesk", sans-serif',
     body: '"Inter", sans-serif',
-    heading: '"Plus Jakarta Sans", sans-serif',
```

Comment:
@nevendyulgerov, 
Why are you putting FK Grotesk back? The Plus Jakarta Sans is the new headline purpose font. The PR #73 made the replacement.

Reply:
I did this because most of the elements on the home page (and other pages) use Grotesk font, so it's less work applying that font globally instead of doing it for each component.

Still, I'll revert this one and tackle the fonts for each of those components.

Reply:
But we removed the FK Grotesk (it may be in the browser cache, but eventually will be gone), so it would fall back to sans-serif. These two are the new fonts, as explained in Figma in the `about` block. Change just the bits your issue covers.

Reply:
Oh! I see what you mean now. yeah, just let's keep the new ones. As I understood from the about-section, Jakarta with different weights is for the heading and button and the body would use the Inter.